src/services/notification/Email/SessionTransferEmail/send.js

```diff
@@ -2,13 +2,13 @@ const sendEmail = require("../sendEmail");
 const { FROM } = require("../../../../constants/email");
 
 
-const send = ({to},{userData,oldSessionData,classData,newSessionData}) => {
+const send = ({to},{userData,businessSessionData,businessClassData,newSession}) => {
```

Comment:
Please comment why should we rename this?

Reply:
argument has to match with the parameters .